bitnami/rabbitmq/templates/NOTES.txt

```diff
@@ -60,10 +60,10 @@ To Access the RabbitMQ Management interface:
 
 {{- if .Values.metrics.enabled }}
 
-To access the RabbitMQ Prometheus metrics, get the RabbitMQ Prometheus exporter URL by running:
+To access the RabbitMQ Prometheus metrics, get the RabbitMQ Prometheus URL by running:
 
-    echo "Prometheus Metrics URL: http://127.0.0.1:{{ .Values.metrics.port }}/metrics"
-    kubectl port-forward --namespace {{ .Release.Namespace }} {{ template "rabbitmq.fullname" . }}-0 {{ .Values.metrics.port }}:{{ .Values.metrics.port }}
+    echo "Prometheus Metrics URL: http://127.0.0.1:15692/metrics"
+    kubectl port-forward --namespace {{ .Release.Namespace }} {{ template "rabbitmq.fullname" . }}-0 15692:15692
```

Comment:
Can we parametrize this port ?

Reply:
It seems it is possible: https://github.com/rabbitmq/rabbitmq-prometheus#configuration